e2e-cypress/files/README.md

```diff
@@ -43,6 +43,34 @@ In order to generate one xml report per test type (installation, integration and
 With Cypress 12 version is now available `cy.origin()` that allows you to handle redirections. This funcionality eases the login handling.
 See `./support/e2e.ts` for a generic login example.
 
+## Cypress Cloud
```

Comment:
I would move the whole section into the documentation of cypress (https://github.com/opendevstack/ods-quickstarters/blob/master/docs/modules/quickstarters/pages/e2e-cypress.adoc). The focus of the `README.md` should be more on getting started. Maybe only leave a short note about general support for Cypress Cloud with a link to the documentation.

Reply:
Done ;)